constants/constants.ts

```diff
@@ -0,0 +1,26 @@
+export const HOME_LOCATION = '/';
+export const FOLDER_LOCATION = '/folder';
+export const SHARED_LOCATION = '/shared';
```

Comment:
💬  how about 
```suggestion
export const ROUTE_PATHS = {
  home: '/',
  folder: '/folder',
  shared: '/shared'
}
```

👀👀👀